modules/ContextUtils.js

```diff
@@ -14,13 +15,15 @@ function makeContextName(name) {
   return `@@contextSubscriber/${name}`
 }
 
+const prefixUnsafeLifecycleMethods = parseFloat(React.version) >= 16.3
```

Comment:
@timdorr as I mentioned over in https://github.com/reduxjs/react-redux/pull/1383#discussion_r329770435 , this parses the latest version of React (16.10.1) as 16.1, so this is broken.